com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -124,6 +124,8 @@ internal struct AgentParameters
         /// </summary>
         public int TeamId
         {
+            // TODO should we remove this and BehaviorName properties,
```

Comment:
Not sure if we need these in the Agent now that the BehaviorParameters are accessible again.

Reply:
Don't need it. Please remove.